dwave/gate/simulator/simulator.pyx

```diff
@@ -168,7 +170,8 @@ def _measure(op, state, targets):
     op._measured_state = state.copy()
 
     for idx, t in enumerate(targets):
-        m = _sample(t, state)
+        # m = _sample(t, state)
+        m = _fast_sample(t, state)
```

Comment:
I feel like we should just remove the old `_sample` function, and rename the `_fast_sample` to `_sample`, just replacing it. There's really no need to keep a slow sampling method anyway, right?